container/container_unix.go

```diff
@@ -147,7 +146,7 @@ func (container *Container) CopyImagePathContent(v volume.Volume, destination st
 			logrus.Warnf("error while unmounting volume %s: %v", v.Name(), err)
 		}
 	}()
-	if err := label.Relabel(path, container.MountLabel, true); err != nil && err != unix.ENOTSUP {
+	if err := label.Relabel(path, container.MountLabel, true); err != nil && !errors.Is(err, syscall.ENOTSUP) {
```

Comment:
Note; I picked `syscall` here, to be consistent with `ignoreUnsupportedXAttrs`. Both `unix.` and `syscall.` work (same error type under the hood)